app/src/main/java/cs465/illinois/edu/dogonthequad/DogOwnerProfileActivity.java

```diff
@@ -33,21 +35,16 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     private void prepareListData() {
+        ArrayList<Dog> dogs = API.getDogs();
         listDataHeader = new ArrayList<String>();
         listDataChild = new HashMap<String, List<String>>();
 
-        listDataHeader.add("Betty");
-        listDataHeader.add("Rover");
+        listDataHeader.add(dogs.get(0).getName());
```

Comment:
should this be converted to some kind of for loop to support more than a fixed number of 2 dogs? Something like:
```JAVA
for (Dog dog : dogs) {
  listDataHeader.add(dog.getName());
  listDataChild.put(dog.getName(), dog.getBadges());
}
```